pyiron_base/jobs/job/generic.py

```diff
@@ -466,6 +476,10 @@ def copy(self):
         _job_reload_after_copy(
             job=copied_self, delete_file_after_copy=delete_file_after_copy
         )
+
+        # Copy executor - it cannot be copied and is just linked instead
+        if self.server.executor is not None:
```

Comment:
Probably doesn't make sense to copy it.  I guess we are mostly copying jobs to use them as reference for some master or move them to other projects.  In neither case we to "duplicate" a result so to speak.

Reply:
While it is technically no problem to copy the future object, we have to ask ourselves what would this mean. The future is only used while the job is executed, copying a job during execution sounds very dangerous to me. If we want to be extra careful we can check if a future exists and then raise a `ValueError`, unless the future is `done()`.

Reply:
> If we want to be extra careful we can check if a future exists and then raise a `ValueError`, unless the future is `done()`.

👍 I like this

Reply:
> The future is only used while the job is executed,

In this case, should we register a callback or otherwise clean up `server.future` when it's finished successfully?

Reply:
The issue with the call back is we do not want the future to be removed when the job is finished but rather we want it to be called when the user recognized the job is finished, so I currently do not see a way how to achieve this with a call back and I would just keep the Future object active. We could even go one step further and if the job is finished, always set a future which is `done`.

Reply:
I see your point about the callback. Also even in the example the future is used post-facto. I'll add a suggestion for the check on copying.